lib/pipeline-watcher/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './watcher-handler';
```

Comment:
I don't think we should export the handler, just the construct. If you want to import it in your unit test you can just use '../lib/pipeline-watcher/watcher-handler'.